share/functions/vi_increment_decrement.fish

```diff
@@ -0,0 +1,34 @@
+alias vi_dec 'vi_increment_decrement subtract'
+alias vi_inc 'vi_increment_decrement add'
+
+function vi_increment_decrement --description 'increment or decrement numbers with ctrl+x ctrl+a'
```

Comment:
I don't believe that a function should mention key bindings. It can hint at defaults, but that is about it.